test/built-ins/Array/prototype/at/coerced-index-resize.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.at
+description: >
+  Array.p.at behaves correctly on TypedArrays backed by resizable buffers when
+  the TypedArray is resized during parameter conversion
+includes: [resizableArrayBufferUtils.js]
+features: [resizable-arraybuffer]
+---*/
+
+function ArrayAtHelper(ta, index) {
+  const result = Array.prototype.at.call(ta, index);
+  return Convert(result);
+}
```

Comment:
```suggestion
function ArrayAtHelper(ta, index) {
  return Array.prototype.at.call(ta, index);
}
```
In this test we can probably skip `Convert()` - it's not necessary to convert bigints back to numbers, because we are only checking that the value returned from `at()` is not `undefined`, anyway.